templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit_master.html.twig

```diff
@@ -68,7 +68,7 @@
                 {% endblock %}
             {% endif %}
 
-            {% if hasProcedureUserRestrictedAccess %}
+            {% if hasProcedureUserRestrictedAccess and hasPermission('feature_procedure_user_restrict_access_edit') %}
```

Comment:
When I can assume, that the permission is set proper in all the projects, this is legit